config/kubernetes/sig-release/teams.yaml

```diff
@@ -246,32 +233,39 @@ teams:
            and can be used as a notification group.
           Remove org members who are not current Release Team Leads.
         members:
-        - guineveresaenger # 1.16 RT Lead Shadow
-        - jeefy # 1.16 RT Lead Shadow
-        - lachie83 # 1.16 RT Lead
-        - nikopen # 1.16 RT Lead Shadow
+        - alejandrox1 # 1.17 RT Lead Shadow
+        - guineveresaenger # 1.17 RT Lead
+        - jeefy # 1.17 RT Lead Shadow
+        - mariantalla # 1.17 RT Lead Shadow
         privacy: closed
   sig-release:
     description: SIG Release Members
     maintainers:
     - mrbobbytables
+    - nikhita
```

Comment:
Out of curiosity, how do we decide who gets added to this team? Is it members who have previously participated in some form in the release team? Is this team also pruned periodically/after a release?

Reply:
@nikhita -- SIG Release has the concept of membership --> https://github.com/kubernetes/community/blob/master/sig-release/charter.md#sig-membership

tl;dr Chairs, subproject owners, former RT role leads

We don't currently prune anyone; only add, but members can definitely ask to be removed.